app/components/user-status-modal.hbs

```diff
@@ -0,0 +1,32 @@
+{{#if @showUserStateModal}}
+<div class="overlay">
+    <div class="modal" data-test-modal>
+        <div class="datePickerAndMessageContainer">
+            <label class="input-description" data-test-labelFrom>From</label>
+            <Input name='fromDate' type='date' value={{this.fromDate}} {{on "change" this.updateValue}}
+                data-test-datePickerFrom />
+            {{#if (eq @newStatus 'OOO')}}
+            <label class="input-description" data-test-labelUntil>Until</label>
+            <Input name='untilDate' type='date' value={{this.untilDate}} {{on "change" this.updateValue}}
+                data-test-datePickerUntil />
+            {{/if}}
+            {{#if (not-eq @newStatus 'ACTIVE')}}
+            {{#if (eq @newStatus 'OOO')}}
+            <label class="input-description" data-test-labelReason>Reason</label>
+            {{/if}}
+            {{#if (eq @newStatus 'IDLE')}}
+            <label class="input-description" data-test-labelMissingSkills>Missing Skills</label>
```

Comment:
Let's have a more descriptive label `What skills are you willing to learn?`

Reply:
yeah sounds better